Transactions.md

```diff
@@ -312,9 +312,57 @@ accept_cet_vbytes = Ceil(accept_cet_weight / 4.0) vbytes
 
 TODO
 
-# References
+# Channel
 
-* [Bitcoin-S implementation](https://github.com/bitcoin-s/bitcoin-s/blob/adaptor-dlc/dlc/src/main/scala/org/bitcoins/dlc/builder)
+## Channel Contract Execution Transaction
```

Comment:
This is pretty much the same as we have in [maia](https://github.com/comit-network/maia/blob/main/maia/src/protocol/transactions.rs]. 

We call the `Buffer Transaction` `CommitTransaction` because you _commit_ to the state :)

Reply:
Yeah I remember that's the naming you use. Again I don't mind that much about the naming but what I worry is that it can be confused with the LN `CommitTransaction` especially if we later specified DLC channels working with LN ones.

Reply:
No worries, I just wanted to point it out :) I have no hard feelings about namings in general.